deploy/consts.ts

```diff
@@ -30,7 +30,7 @@ export const L1_ADDRESS_MAP: { [key: number]: { [contractName: string]: string }
     usdc: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
     scrollERC20GatewayRouter: "0xF8B1378579659D8F7EE5f3C929c2f3E332E41Fd6",
     scrollMessengerRelay: "0x6774Bcbd5ceCeF1336b5300fb5186a12DDD8b367",
-    scrollGasPriceOracle: "0x987e300fDfb06093859358522a79098848C33852",
+    scrollGasPriceOracle: "0x0d7E906BD9cAFa154b048cFa766Cc1E54E39AF9B",
```

Comment:
The existing mainnet gas price oracle was migrated in ~February: https://docs.scroll.io/en/developers/scroll-contracts/#advanced-bridge-contracts